frontend/public/components/utils/resource-log.jsx

```diff
@@ -94,7 +111,16 @@ export class ResourceLog extends React.Component {
     return null;
   }
 
+  fetchPodLogLinks() {
+    coFetchJSON('api/kubernetes/apis/console.openshift.io/v1/consolepodloglinks')
```

Comment:
If the CRD does not exist, an error is output to the browser console:
```
GET http://localhost:9000/api/kubernetes/apis/console.openshift.io/v1/consolepodloglinks 404 (Not Found)
```

Reply:
Not great as these are not expected to exist...